JSDemos/Demos/Tabs/Overview/ReactJs/App.js

```diff
@@ -29,10 +29,28 @@ const App = () => {
   const [showNavigation, setShowNavigation] = React.useState(false);
   const [scrollContent, setScrollContent] = React.useState(false);
   const [fullWidth, setFullWidth] = React.useState(false);
+  const [width, setWidth] = React.useState('auto');
   const [rtlEnabled, setRtlEnabled] = React.useState(false);
+  const [selectedIndex1, setSelectedIndex1] = React.useState(0);
+  const [selectedIndex2, setSelectedIndex2] = React.useState(0);
+  const [selectedIndex3, setSelectedIndex3] = React.useState(0);
   const [widgetWrapperClasses, setWidgetWrapperClasses] = React.useState(
     'widget-wrapper widget-wrapper-horizontal',
   );
+  const toggleShouldRestrictWidth = React.useCallback(
+    (shouldRestrictWidth) => {
+      if (shouldRestrictWidth) {
+        setWidgetWrapperClasses(`${widgetWrapperClasses} strict-width`);
```

Comment:
in this case it'll work, but directly using widgetWrapperClasses state instead of the functional form with 'prev' callback is considered not a good practice, to avoid potential issues with asynchronous nature of state updates i propose this
setWidgetWrapperClasses((prevClasses) => `${prevClasses} strict-width`);

[minor] also ,just in case, plz consider checking if class is already present.
setWidgetWrapperClasses((prevClasses) =>
        prevClasses.includes('strict-width')
          ? prevClasses
          : `${prevClasses} strict-width`
      );

Reply:
Yes, it looks better.

Reply:
Fixed in [5b9bd3875eddb26acdca83ce94cfe7a767786467].